model/src/main/kotlin/Project.kt

```diff
@@ -85,6 +85,11 @@ data class Project(
      */
     val vcsProcessed: VcsInfo = vcs.normalize(),
 
+    /**
```

Comment:
Commit message: This should resolve https://github.com/oss-review-toolkit/ort/issues/9443.

Reply:
Is it not sufficient to resolve it from the PR's description ?

Reply:
> Is it not sufficient to resolve it from the PR's description ?

Technically that would suffice to auto-close the issue, but our policy has always been to put this kind of information into the Git commit history.

Reply:
ok, so I will follow the policy. (Sometimes I dislike that it spams referenced issue in case of a rebase workflow. e.g. each force push adds some new reference to the issue)